react-app/src/styles/FeatureDescription.module.css

```diff
@@ -0,0 +1,76 @@
+.container {
```

Comment:
h1 tag containing site.label is visually rendered outside the container
![image](https://github.com/user-attachments/assets/39796bb4-c32f-4a8f-9d3d-2a6a6b6c0225)

Reply:
as you can see the "Go" button also moves outside the container